templates/repo/issue/view_content/pull.tmpl

```diff
@@ -444,52 +496,346 @@
+											{{else}}
+												{{$.i18n.Tr "repo.pulls.rebase_merge_commit_pull_request"}}
+											{{end}}
+										</button>
+										<button class="ui button merge-cancel">
+											{{$.i18n.Tr "cancel"}}
+										</button>
+										{{if .IsPullBranchDeletable}}
+											<div class="ui checkbox ml-2">
+												<input name="delete_branch_after_merge" type="checkbox" {{if $prUnit.PullRequestsConfig.DefaultDeleteBranchAfterMerge}}checked{{end}}>
+												<label>{{$.i18n.Tr "repo.branch.delete" .HeadTarget}}</label>
+											</div>
 										{{end}}
-										{{if eq .MergeStyle "rebase-merge"}}
-											{{$.i18n.Tr "repo.pulls.rebase_merge_commit_pull_request"}}
+									</form>
+								</div>
+							{{end}}
+							{{if $prUnit.PullRequestsConfig.AllowSquash}}
+								<div class="ui form squash-now-fields" style="display: none">
+									<form action="{{.Link}}/merge" method="post">
+										{{.CsrfTokenHtml}}
+										<div class="field">
+											<input type="text" name="merge_title_field" value="{{.Issue.PullRequest.GetDefaultSquashMessage}}">
+										</div>
+										<div class="field">
+											<textarea name="merge_message_field" rows="5" placeholder="{{$.i18n.Tr "repo.editor.commit_message_desc"}}">{{.GetCommitMessages}}Reviewed-on: {{$.Issue.HTMLURL}}&#13;&#10;{{$approvers}}</textarea>
+										</div>
+										<button class="ui red button" type="submit" name="do" value="squash">
+											{{$.i18n.Tr "repo.pulls.squash_merge_pull_request_now"}}
+										</button>
+										<button class="ui button merge-cancel">
+											{{$.i18n.Tr "cancel"}}
+										</button>
+										{{if .IsPullBranchDeletable}}
+											<div class="ui checkbox ml-2">
+												<input name="delete_branch_after_merge" type="checkbox" {{if $prUnit.PullRequestsConfig.DefaultDeleteBranchAfterMerge}}checked{{end}}>
+												<label>{{$.i18n.Tr "repo.branch.delete" .HeadTarget}}</label>
+											</div>
 										{{end}}
-										{{if eq .MergeStyle "squash"}}
-											{{$.i18n.Tr "repo.pulls.squash_merge_pull_request"}}
+									</form>
+								</div>
+								<div class="ui form squash-fields" style="display: none">
+									<form action="{{.Link}}/merge" method="post">
+										{{.CsrfTokenHtml}}
+										{{if and $statusChecksPending (not .HasPendingPullRequestMerge)}}
+											<input type="hidden" name="merge_when_checks_succeed" value="1"/>
 										{{end}}
-										{{if eq .MergeStyle "manually-merged"}}
-											{{$.i18n.Tr "repo.pulls.merge_manually"}}
+										<div class="field">
+											<input type="text" name="merge_title_field" value="{{.Issue.PullRequest.GetDefaultSquashMessage}}">
+										</div>
+										<div class="field">
+											<textarea name="merge_message_field" rows="5" placeholder="{{$.i18n.Tr "repo.editor.commit_message_desc"}}">{{.GetCommitMessages}}Reviewed-on: {{$.Issue.HTMLURL}}&#13;&#10;{{$approvers}}</textarea>
```

Comment:
I am absolutely astonished by now.
Please don't tell me there will be a sixth.

Reply:
no idea, relics from the past :dagger:

Reply:
yep that's the part I need help ... this template is just too bloated!!!